3-UsingAPIs/Week2/homework/ex1-programmerFun/index.js

```
@@ -17,29 +17,39 @@ Full description at: https://github.com/HackYourFuture/Homework/blob/main/3-Usin
    url with `.shx`. There is no server at the modified url, therefore this 
    should result in a network (DNS) error.
 ------------------------------------------------------------------------------*/
-function requestData(url) {
-  // TODO return a promise using `fetch()`
+async function requestData(url) {
+  try {
+    const response = await fetch(url);
+    if (!response.ok) {
+      throw new Error(response.statusText);
+    }
+    const data = await response.json();
+    console.log(data);
+    return data;
+  } catch {}
```

What is the purpose of this empty catch block? In fact , the whole try/catch structure doesn't serve any purpose here because it is not catching any errors.